packages/utils/interactive-hooks/src/useKeyProps.ts

```diff
@@ -1,13 +1,16 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { KeyPressEvent } from './Pressability/CoreEventTypes';
+import { IHandledKeyboardEvent } from '@office-iss/react-native-win32';
 
 export type KeyCallback = (args?: KeyPressEvent) => void;
 
 export type KeyPressProps = {
+  onKeyDown?: KeyCallback;
+  validKeysDown?: string[]; // macOS
+  keyDownEvents?: IHandledKeyboardEvent[]; // win32
   onKeyUp?: KeyCallback;
   validKeysUp?: string[];
-  onKeyDown?: KeyCallback;
-  validKeysDown?: string[];
+  keyUpEvents?: IHandledKeyboardEvent[]; // win32
```

Comment:
Is the comment here supposed to say macos?

Reply:
Oh, nvm. I think I understand

Reply:
I did miss a macOS for validKeysUp. so that's fair.

What's fun is windows has slightly different type for their `keyDownEvents` prop. I intentionally ommitted adding that to not deal with that for now :D